javascript-practice/src/js/controllers/dashboardController.js

```diff
@@ -24,13 +24,15 @@ export default class DashboardController {
     this.renderTableListUsers();
   }
 
+  //The checkAccess function is used to check user access by checking the roles saved in localStorage.
```

Comment:
Check comment format for function here: https://javascript.info/comments#good-comments

Reply:
No need to repeat function name